src/models/newsletters.ts

```diff
@@ -6,29 +6,44 @@ const NewsletterIllustrationType = t.type({
 	circle: t.string,
 });
 
-const EmailEmbedType = t.type({
+const baseEmailEmbedModel = {
 	name: NonEmptyString,
 	title: NonEmptyString,
-	description: NonEmptyString,
+	description: optional(t.string),
 	successHeadline: NonEmptyString,
 	successDescription: NonEmptyString,
 	hexCode: NonEmptyString,
 	imageUrl: optional(t.string), //deprecated
+};
+
+const EmailEmbedType = t.type({
+	...baseEmailEmbedModel,
 });
 
-export const EmailNewsletterType = t.type({
+// the value for EmailNewsletter.emailEmbed.description comes from the
+// "mailDescription" cell or if that if empty, the "description" cell.
+// As EmailNewsletter.description (from the "description" cell) is
+// NonEmptyString for a CurrentEmailNewsletter, so should
+// EmailNewsletter.emailEmbed.description.
```

Comment:
I'm not sure I understand this comment entirely

Reply:
I'll re-write that! What I was trying to say was that on a cancelled newsletter `emailEmbed.description` can be empty, because the fallback value for it come from `description`, which can be empty on a cancelled newsletter (but must be non-empty for a current newsletter).